language/diem-tools/diem-validator-interface/src/lib.rs

```diff
@@ -53,6 +50,48 @@ pub trait DiemValidatorInterface {
         }
         Ok(acc)
     }
+
+    fn get_key_accounts_by_version(
```

Comment:
Nit: `get_admin_accounts`, `get_governance_accounts` or similar might be better? The term "key" is so overloaded that I'd prefer not to use it...

Reply:
Also, a comment saying what this does seems useful.

Reply:
Fixed!